From Viral.Mehta@lntinfotech.com  Thu Apr 29 13:05:59 2010
From: Viral Mehta <Viral.Mehta@lntinfotech.com>
Date: Tue, 6 Apr 2010 11:51:00 +0530
Subject: USB: omap: switch to subsys_initcall for isp1301 transceiver
Message-ID: <70376CA23424B34D86F1C7DE6B9973430254343A39@VSHINMSMBX01.vshodc.lntinfotech.com>


isp1301 transceiver driver init should be done before we do ohci omap init

Signed-off-by: Viral Mehta <viral.mehta@lntinfotech.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/otg/isp1301_omap.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/otg/isp1301_omap.c
+++ b/drivers/usb/otg/isp1301_omap.c
@@ -1654,7 +1654,7 @@ static int __init isp_init(void)
 {
 	return i2c_add_driver(&isp1301_driver);
 }
-module_init(isp_init);
+subsys_initcall(isp_init);
 
 static void __exit isp_exit(void)
 {
